test/user.test.js

```diff
@@ -67,6 +68,26 @@ describe('User', function() {
       });
     });
 
+   it('Create a new user (email case-sensitivity off)', function(done) {
+      User.settings.caseSensitiveEmail = false;
+      User.create({email: 'F@b.com', password: 'bar'}, function(err, user) {
+        assert(!err);
```

Comment:
> I followed the same conventions at the other tests in that describe block.

It's an old convention that I would like us to eventually get rid of.

> Should I replace assert(!err) in all cases then?

I don't mind either. However, if you decide to fix existing tests, then make it in a standalone commit. This will make future inspection (code archaeology) much easier.